packages/env/src/index.ts

```diff
@@ -33,6 +33,23 @@ export function createSHA256(): SHA256 {
   };
 }
 
+export function randomUUIDFromString(input: string) {
```

Comment:
1. the idea of env is to provide env specific implementations, here we're only providing the node-specific, so the non-node is missing
2. if we use `createSHA256()`, then we can move this to core (in `utils/uuid.ts` file) as createSHA256 is provided for all envs. 

isn't the result always the same? then it shouldn't be `random...`:
```suggestion
export function UUIDFromString(input: string) {
```